crates/opencascade/src/primitives/face.rs

```diff
@@ -18,20 +18,25 @@ impl AsRef<Face> for Face {
 }
 
 impl Face {
-    fn from_make_face(make_face: UniquePtr<ffi::BRepBuilderAPI_MakeFace>) -> Self {
-        let face = make_face.Face();
+    pub(crate) fn from_face(face: &ffi::TopoDS_Face) -> Self {
         let inner = ffi::TopoDS_Face_to_owned(face);
 
         Self { inner }
     }
 
+    fn from_make_face(make_face: UniquePtr<ffi::BRepBuilderAPI_MakeFace>) -> Self {
+        Self::from_face(make_face.Face())
+    }
+
+    #[must_use]
```

Comment:
This is fine to have but we can probably relax the usage of `#[must_use]` for constructors, since callers don't have an instance of the struct to mutate.

Basically any method with `&self` which sounds like it might mutate `self`, but actually returns a new `Shape` will benefit from having `#[must_use]`.

Reply:
@julianschuler I pushed a few changes to address this comment. Once it passes CI I'll go ahead and merge it :)